lib/policy_machine.rb

```diff
@@ -69,7 +69,7 @@ def add_association(user_attribute_pe, operation_set, object_attribute_pe)
   # TODO: Parallelize the two component checks
   def is_privilege?(user_or_attribute, operation, object_or_attribute, options = {})
     is_privilege_ignoring_prohibitions?(user_or_attribute, operation, object_or_attribute, options) &&
-      !is_privilege_ignoring_prohibitions?(user_or_attribute, PM::Prohibition.on(operation), object_or_attribute, options)
+      (options[:ignore_prohibitions] || !is_privilege_ignoring_prohibitions?(user_or_attribute, PM::Prohibition.on(operation), object_or_attribute, options))
```

Comment:
Oh, right, no, that's to support the default implementation that uses is_privilege? (line 204). Easiest way to have it support ignore_prohibitions was to pass it through to is_privilege?.